From b4fa4bbee126ded52bb806ffb60c61448924b9a6 Mon Sep 17 00:00:00 2001
From: Graham Christensen <graham@grahamc.com>
Date: Tue, 21 Nov 2017 20:20:46 -0500
Subject: [PATCH] Update daps-init

---
 bin/daps-init | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/bin/daps-init b/bin/daps-init
index 64fa9cb1..fe49e085 100755
--- a/bin/daps-init
+++ b/bin/daps-init
@@ -86,13 +86,7 @@ trap "exit_on_error '\nCaught SIGTERM/SIGINT'" SIGTERM SIGINT
 # show help when called with no arguments
 [[ -z "$1" ]] && usage
 
-# check for xmlstarlet binary
-for _binary in xmlstarlet xml; do
-    XMLSTARLET=$(which $_binary 2>/dev/null)
-    [[ 0 -eq $? ]] && break
-done
-
-[[ -z $XMLSTARLET ]] && exit_on_error "Required package \"xmlstarlet\" is not installed"
+XMLSTARLET="@xmlstarlet@/bin/xmlstarlet"
 
 #----------------
 # Parse the command line arguments
